libs/wallet/src/TransactionUpdate.graphql

```diff
@@ -0,0 +1,93 @@
+fragment TransactionResultEvent on TransactionResult {
+  partyId
+  hash
+  status: status
+  error
+}
+
+fragment WithdrawalEvent on Withdrawal {
+  id
+  withdrawalStatus: status
+  amount
+  asset {
+    id
+    name
+    symbol
+    decimals
+    status
+    source {
+      ... on ERC20 {
+        contractAddress
+      }
+    }
+  }
+  createdTimestamp
+  withdrawnTimestamp
+  txHash
+  details {
+    ... on Erc20WithdrawalDetails {
+      receiverAddress
+    }
+  }
+  pendingOnForeignChain @client
+}
+
+fragment OrderEvent on Order {
+  type
+  id
+  orderStatus: status
+  rejectionReason
+  createdAt
+  size
+  price
+  timeInForce
+  expiresAt
+  side
+  market {
+    id
+    decimalPlaces
+    positionDecimalPlaces
+    tradableInstrument {
+      instrument {
+        name
+      }
+    }
+  }
+}
+
+fragment DepositEvent on Deposit {
+  id
+  depositStatus: status
+  amount
+  asset {
+    id
+    symbol
+    decimals
+  }
+  createdTimestamp
+  creditedTimestamp
+  txHash
+}
+
+subscription TransactionUpdate($partyId: ID!) {
+  busEvents(
+    partyId: $partyId
+    batchSize: 0
+    types: [Withdrawal, Deposit, TransactionResult, Order]
+  ) {
+    event {
+      ... on Deposit {
```

Comment:
There are other transactions that other apps will depend on (eg. proposals on the governance app). It seems a little heavy handed to force an app to subscribe to all types of updates, even if it doesn't care about them.

Can we decouple this in someway that allows the app to specify which types of transaction to listen for?